onnxruntime/automl_ops/cpu/datetime_transformer.cc

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "core/common/common.h"
+#include "core/framework/data_types.h"
+#include "core/framework/op_kernel.h"
+
+#include "core/automl/featurizers/src/FeaturizerPrep/Featurizers/DateTimeFeaturizer.h"
+
+namespace dtf = Microsoft::Featurizer::DateTimeFeaturizer;
+
+namespace onnxruntime {
+namespace automl {
+
+class DateTimeTransformer final : public OpKernel {
+ public:
+  explicit DateTimeTransformer(const OpKernelInfo& info) : OpKernel(info) {}
+  Status Compute(OpKernelContext* context) const override;
+};
+
+Status DateTimeTransformer::Compute(OpKernelContext* ctx) const {
+  Status s;
+  auto input_tensor = ctx->Input<Tensor>(0);
+  dtf::TimePoint* output = ctx->Output<dtf::TimePoint>(0);
+
+  int64_t tp = *input_tensor->Data<int64_t>();
```

Comment:
Is this supposed to be a transformer that transforms a tensor (not necessarily a scalar) of ints to a tensor/array of time_points? Or, is it supposed to be restricted to the case where the input is a scalar? Assuming that automl needs to work with batches of inputs, I would have expected support for a tensor of ints, not just a scalar.

Reply:
The answer to this is no. This transformer transforms a single timepoint into a Featurizer specific type. We may come to batches. This kernel is very experimental to verify that we can share the code, create models on a .MET side and run them.